### PR TITLE
move minGCAge to ImageGCPolicy and make it configurable

### PR DESCRIPTION
**Pull request by mqliang**
Depending on https://github.com/kubernetes/kubernetes/pull/18820
@vishh

### CODE REVIEW

**Comment by mqliang** [comment]
@k8s-bot test this please

**Comment by mqliang** [comment]
@vishh

File: cmd/kubelet/app/options/options.go [hunk 1]
```diff
@@ -180,6 +181,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.NodeStatusUpdateFrequency.Duration, "node-status-update-frequency", s.NodeStatusUpdateFrequency.Duration, "Specifies how often kubelet posts node status to master. Note: be cautious when changing the constant, it must work with nodeMonitorGracePeriod in nodecontroller. Default: 10s")
 	bindableNodeLabels := util.ConfigurationMap(s.NodeLabels)
 	fs.Var(&bindableNodeLabels, "node-labels", "<Warning: Alpha feature> Labels to add when registering the node in the cluster.  Labels must are key=value pairs seperated by ','.")
+	fs.DurationVar(&s.ImageMinimumGCAge.Duration, "minimum-image-ttl-duration", s.ImageMinimumGCAge.Duration, "Minimum age for a unused image before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'")
```
Comment (vishh) [inline cmd/kubelet/app/options/options.go]: Let's set the default to 2 minutes so that things work sanely off the box.

**Comment by vishh** [comment]
Just one nit. Otherwise LGTM

**Comment by mqliang** [comment]
@vish Rebased and set the default to 2 minutes. PTAL

BTW, I think we should rename the `MinimumGCAge`(https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L100) as `ContainerMinimumGCAge`, which is much more readable. Thought?

**Comment by vishh** [comment]
@mqliang: If you fix the one pending comment, this PR can be merged :+1:

**Comment by mqliang** [comment]
@vishh I have set the default to 2 minutes and rebased. Could you please reapply LGTM tag?

**Comment by mqliang** [comment]
@k8s-bot unit test this please.

**Comment by mqliang** [comment]
@vishh Rebased. Could you please kindly reapply LGTM?

**Comment by mqliang** [comment]
@vishh Rebased. Please reapply LGTM. BTW, could this be manually merged if all test passed? Since the auto generated file will conflict almost every day.

**Comment by vishh** [comment]
@mqliang: Bumped up the priority. Hopefully this gets merged soon.

**Pull request merged**